### PR TITLE
#1 Create solution structure.

### LINKED ISSUE #1: [1] Create solution structure.
It should be our common project structure with entry point based on ASP.NET Core template (`webapi`). Core project, Database with 3 related projects such as Models, Repositories and Context.

### PR DESCRIPTION
**Pull request by DeiLux**

### CODE REVIEW

File: src/OverbeeredTGNewsAggregatorBot.Server/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "dotnetRunMessages": "true",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "applicationUrl": "https://localhost:5001;http://localhost:5000",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
+  }
+}
```
Comment (q00Dree) [inline src/OverbeeredTGNewsAggregatorBot.Server/Properties/launchSettings.json]:
```suggestion
{
  "$schema": "http://json.schemastore.org/launchsettings.json",
  "profiles": {
    "OverbeeredTGNewsAggregatorBot.Server": {
      "commandName": "Project",
      "launchBrowser": true,
      "launchUrl": "swagger",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      },
      "applicationUrl": "http://localhost:5000"
    }
  }
}
```

File: src/OverbeeredTGNewsAggregatorBot.Server/OverbeeredTGNewsAggregatorBot.Server.csproj [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.6.3" />
+  </ItemGroup>
```
Comment (q00Dree) [inline src/OverbeeredTGNewsAggregatorBot.Server/OverbeeredTGNewsAggregatorBot.Server.csproj]: Be sure, If you create project with -f net5.0 and then edit it to net6.0 you need to update Nuget depencies. Current **Swashbuckle.AspNetCore** version is **6.2.3**

Comment (q00Dree) [reply src/OverbeeredTGNewsAggregatorBot.Server/OverbeeredTGNewsAggregatorBot.Server.csproj]: You can check updates of nugets for whole solution.

**Step 1**
image

**Step 2**
image

Comment (DeiLux) [reply src/OverbeeredTGNewsAggregatorBot.Server/OverbeeredTGNewsAggregatorBot.Server.csproj]: он только со 2 раза показывает обновления

Comment (q00Dree) [reply src/OverbeeredTGNewsAggregatorBot.Server/OverbeeredTGNewsAggregatorBot.Server.csproj]: So what?

Comment (DeiLux) [reply src/OverbeeredTGNewsAggregatorBot.Server/OverbeeredTGNewsAggregatorBot.Server.csproj]: все ебать

File: src/OverbeeredTgNewsBot.Server/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "OverbeeredTGNewsAggregatorBot.Server": {
```
Comment (q00Dree) [inline src/OverbeeredTgNewsBot.Server/Properties/launchSettings.json]: Change it on `OverbeeredTgNewsBot.Server`. 
**TG** is relevant anymore.

**Comment by q00Dree** [comment]
I can't get, why you reduced name from  **OverbeeredTgNewsAggregatorBot** to **OverbeeredTgNewsBot**? In my opinion **aggregator** is a meaningful thing in project name... And why you place **Overbeered** here and in other projects? It would better to place **Fishie** as unique name of the project.

**Comment by DeiLux** [comment]
не понял насчет  "Fishie", что имеешь ввиду ?

**Comment by q00Dree** [comment]
> не понял насчет "Fishie", что имеешь ввиду ?

You should rename it from **OverbeeredTgNewsBot** to **Fishie**, as we do in **Froggie** project.
After that, our projects will be named as **Fishie.Core** and so on.

**Comment by q00Dree** [comment]
Status, please @gena56rus1 
Did you rename .sln file?

**Comment by DeiLux** [comment]
yes

**Pull request merged**